pyproject.toml

```diff
@@ -27,6 +27,9 @@ dev = [
 demo = [
     "notebook>=7.3.3",
 ]
+maxar = [
+    "pystac-client>=0.8.6,<1",
+]
```

Comment:
Maybe just consolidate to a single `notebook` group? Just my experience, but I haven't found it that useful to make a bunch of little groups.

Reply:
I was thinking ahead for having an ingest recurring task like "sync Maxar catalog" that would include this package feature

Looking back though, I'm not sure we really need `pystac-client`? I think since it's a static catalog that `pystac` itself can do what we need in terms of resolving Catalog->Collections->Items 🤔

Reply:
+1 we don't need `pystac-client` and can just use `pystac` 🎉. I'll remove this from dependencies entirely